test/CSModule.t.sol

```diff
@@ -5744,6 +5744,45 @@ contract CsmSettleELRewardsStealingPenaltyBasic is CSMCommon {
         assertEq(secondLock.retentionUntil, 0);
     }
 
+    function test_settleELRewardsStealingPenalty_withDuplicates() public {
+        uint256 firstNoId = createNodeOperator();
+        uint256[] memory idsToSettle = new uint256[](3);
+        idsToSettle[0] = firstNoId;
+        idsToSettle[2] = firstNoId;
```

Comment:
technically there are three zero values in the array

Reply:
fixed